R/g_boxplot.R

```diff
@@ -10,35 +10,35 @@
 #' of the box indicate the degree of dispersion (spread) and skewness in the
 #' data, and show outliers. In addition to the points themselves, they allow one
 #' to visually estimate various L-estimators, notably the interquartile range,
-#' midhinge, range, mid-range, and trimean.
+#' `midhinge`, range, mid-range, and `trimean`.
 #'
-#' @param data ADaM structured analysis laboratory data frame e.g. ADLB.
-#' @param biomarker biomarker to visualize e.g. IGG.
-#' @param param_var name of variable containing biomarker codes e.g. PARAMCD.
+#' @param data `ADaM` structured analysis laboratory data frame e.g. `ADLB`.
+#' @param biomarker biomarker to visualize e.g. `IGG`.
+#' @param param_var name of variable containing biomarker codes e.g. `PARAMCD`.
 #' @param yaxis_var name of variable containing biomarker results displayed on
-#'   Y-axis e.g. AVAL.
-#' @param trt_group name of variable representing treatment trt_group e.g. ARM.
-#' @param loq_flag_var  name of variable containing LOQ flag e.g. LOQFL.
-#' @param loq_legend `logical` whether to include LoQ legend.
+#'   Y-axis e.g. `AVAL`.
+#' @param trt_group name of variable representing treatment `trt_group` e.g. `ARM`.
+#' @param loq_flag_var  name of variable containing `LOQ` flag e.g. `LOQFL`.
+#' @param loq_legend `logical` whether to include `LoQ` legend.
 #' @param unit biomarker unit label e.g. (U/L)
-#' @param color_manual vector of color for trt_group
-#' @param shape_manual vector of shapes (used with loq_flag_var)
+#' @param color_manual vector of color for `trt_group`
+#' @param shape_manual vector of shapes (used with `loq_flag_var`)
 #' @param box add boxes to the plot (boolean)
 #' @param ylim ('numeric vector') optional, a vector of length 2 to specify the minimum and maximum of the y-axis
 #'   if the default limits are not suitable.
-#' @param facet_var variable to facet the plot by, or "None" if no faceting
+#' @param facet_var variable to facet the plot by, or `"None"` if no faceting
```

Comment:
why would "None" be printed as code?

Reply:
IMHO the quotation indicates character type of object whereas no-quotation indicates unknown object (incl. function), e.g. let's have the following `foo_factory <- function(foo_fun = foo, name = "test") {...}` and when documenting it I can state that `foo_fun` takes `foo` as a default (i.e. no quote) and the default for `name` argument is `"test"` (quoted). So I do agree with this change, but... (see below)

Yet another thing is whether this is correct. Something close to `None` looked very pythonish and I would double-check if this should have been state as being empty.

Reply:
Fair enough, code it is.

`if (facet_var != "None" & facet_var %in% names(data)) {` suggests that the "None" literal is just a way to specify one doesn't want faceting. Although I I'm not entirely sure why it is only invoked `if (!is.null(facet_var)) {`.